src/Apps/gEvGen.cxx

```diff
@@ -649,84 +652,55 @@ void GetCommandLineArgs(int argc, char ** argv)
   RunOpt::Instance()->EnableBareXSecPreCalc(true);
   RunOpt::Instance()->ReadFromCommandLine(argc,argv);
 
-  // Parse run options for this app
-
-  CmdLnArgParser parser(argc,argv);
-
-  // help?
-  bool help = parser.OptionExists('h');
-  if(help) {
-      PrintSyntax();
-      exit(0);
-  }
-
-  // number of events
-  if( parser.OptionExists('n') ) {
-    LOG("gevgen", pINFO) << "Reading number of events to generate";
-    gOptNevents = parser.ArgAsInt('n');
-  } else {
-    LOG("gevgen", pINFO)
-       << "Unspecified number of events to generate - Using default";
-    gOptNevents = kDefOptNevents;
-  }
-
-  // run number
-  if( parser.OptionExists('r') ) {
-    LOG("gevgen", pINFO) << "Reading MC run number";
-    gOptRunNu = parser.ArgAsLong('r');
-  } else {
-    LOG("gevgen", pINFO) << "Unspecified run number - Using default";
-    gOptRunNu = kDefOptRunNu;
+  // temporarys
```

Comment:
spelling "temporaries".

This is an interesting addition / upgrade for command line handling.

Reply:
is this just about the typo?